saos-common/src/test/java/pl/edu/icm/saos/common/json/JsonUtilsTest.java

```diff
@@ -30,24 +39,30 @@ public void before() throws JsonParseException, IOException {
     
     
     @Test
-    public void nextNode_SingleElement() throws JsonParseException, IOException {
+    public void nextObject_ClassObject() throws JsonParseException, IOException {
         
         // Given
-        String content = normalizeJson("{bre: 'sss', arr:['1112', 'abc']}");
+        String content = normalizeJson("{name: 'sss', somelist:['1112', 'abc']}");
         
         JsonParser jsonParser = jsonFactory.createParser(content);
         
         
         // Execute
-        String nodeStr = JsonUtils.nextNode(jsonParser);
+        JsonItem jsonItem = jsonUtils.nextJsonObject(jsonParser, JsonItem.class);
```

Comment:
Because it concerns only one case I am not going to change it (especially here the 3 assertions are not too bad because thanks to that we can find the wrong assertion easier).